www/app/app.ts

```diff
@@ -156,6 +158,23 @@ export class App {
     }
   }
 
+  private maybeShowPrivacyView() {
+    let privacyAcked = false;
+    try {
+      privacyAcked = this.settings.get(SettingsKey.PRIVACY_ACK) === 'true';
+    } catch (e) {
+      console.error(`could not read privacy acknowledgement setting, assuming not akcnowledged`);
+    }
+    this.rootEl.$.serversView.hidden = !privacyAcked;
+    this.rootEl.$.privacyView.hidden = privacyAcked;
+  }
+
+  private privacyAcked() {
```

Comment:
Let's use a more explicit name, like `acceptPrivacyTerms()`.

More on naming, I find it better to name those event handlers as commands by the user, rather than past-tense phrases (past tense is great for events). One way to think about it is: imagine a hypothetical situation where you want to simulate a user using the app. You could call `app.acceptPrivacyTerms()` and it would make sense as a user action.

Reply:
Done in a2ecc48. Thank you for the explanation.